contents/blog/a-non-coders-thoughts-on-everybody-codes-culture.md

```diff
@@ -0,0 +1,39 @@
+---
+date: 2021-06-14
+title: A non-coders thoughts on an everybody codes culture
+author: joe-martin
+rootPage: /blog 
+sidebar: Blog 
+showTitle: true 
+hideAnchor: true 
+---
+By: [Joe Martin](https://www.linkedin.com/in/joemartinwords/), Product Marketing
+
+One of PostHog’s core values is that everybody codes. That doesn’t mean everyone needs previous experience as a software developer, but we do encourage everyone to learn and practice the basics of shipping, no matter what their role is. 
+
+When we’re hiring developers this isn’t an issue. However it can be intimidating for candidates who don’t have developer experience, even if they are otherwise technically literate. This is because our tools are centralized around GitHub, so even writing this blogpost involves knowing how to submit Pull Requests. 
+
+As someone who recently joined PostHog, there were three things which helped set me up for success when using these tools for the first time. 
+
+## Getting started early
+There’s a lot to be done when joining a new company and intimidating tasks can end up deprioritized - especially if they’d normally fall outside your remit. 
+
+One of the best things PostHog’s team did was push against this and get me started with coding as soon as possible. This included sharing videos about GitHub fundamentals before I started and setting up tools such as Microsoft Visual Studio with me. It was clear this needed to be a point of focus. 
```

Comment:
Nit: "(MS) Visual Studio Code".

- Microsoft "Visual Studio" is a hardcore development environment from the 1990s 
- Microsoft "Visual Studio Code" or "vscode" is a new editor built with web technologies that we use to code PostHog.

![image](https://user-images.githubusercontent.com/53387/121906585-8abf1b00-cd2b-11eb-848a-5affb495683f.png)

Reply:
You have missed your calling as a proofreader <3

Reply:
You mean narrowly escaped? :D